assets/src/components/v2/disruption_diagram/disruption_diagram.tsx

```diff
@@ -764,8 +758,9 @@ const DisruptionDiagram: ComponentType<DisruptionDiagramData> = (props) => {
           <g
             id="alert-emphasis"
             transform={`translate(0, ${
-              EMPHASIS_HEIGHT / 2 +
-              (EMPHASIS_HEIGHT - MAX_ICON_HEIGHT) * scaleFactor
+              40 // Half the height of the emphasis icon
+              + 24 * scaleFactor // Half the height of the largest icon, "you are here" octagon
+              + 8 * scaleFactor // Emphasis padding
```

Comment:
Any reason for switching to number literals here? It looks like the `EMPHASIS_HEIGHT` and `MAX_ICON_HEIGHT` constants are still defined.

Oh, it looks like the new constant values are slightly different from this now. EMPHASIS_HEIGHT (80) - MAX_ICON_HEIGHT (64) = 16, not 24. So is that why, maybe?

(The added comments do help a lot, either way)

Reply:
Good point... I started realizing that the things I thought were constant actually varied in different situations 😓  So the max icon height for a diamond _endpoint_ is 64, while the max icon height for a diamond in the _middle slot_ is 48, so I started diverging from the constants..... While the comments are helpful, that is still a bit confusing if someone were to look at this down the road. So I'll try to clean it up a bit